test/SIL.XForge.Tests/Services/UserServiceTests.cs

```diff
@@ -295,9 +295,9 @@ private class TestEnvironment
             public UserService Service { get; }
             public MemoryRepository<UserSecret> UserSecrets { get; }
             public MemoryRealtimeService RealtimeService { get; }
-            public IAuthService AuthService = Substitute.For<IAuthService>();
+            public readonly IAuthService AuthService = Substitute.For<IAuthService>();
```

Comment:
How are you deciding which members to mark `readonly`? At first I thought it was the ones that didn't have getters, but `IssuedAt` doesn't follow that pattern.

Reply:
`readonly` means that the variable can only be set in the constructor. It stops other code inadvertently changing the variable to refer to another object and causing unintended side effects.

Reply:
So is there a reason `IssuedAt` wouldn't be marked `readonly` as well? As far as I can tell it is only ever read.

Reply:
Good spotting. Because it had an `=>` assignment, not an `=`. Fixed - thank you! I couldn't find any others like this in the test classes with a quick search.

Reply:
Ah, OK, I didn't even notice it was a method.